tests/integration/iface_admin_state_test.py

```diff
@@ -21,13 +21,19 @@
 
 import libnmstate
 
+from libnmstate.schema import Interface
 from .testlib import assertlib
-from .testlib.statelib import INTERFACES
 
 
 def test_set_a_down_iface_down(eth1_up):
     desired_state = {
-        INTERFACES: [{'name': 'eth1', 'type': 'ethernet', 'state': 'down'}]
+        Interface.KEY: [
+            {
+                Interface.NAME: 'eth1',
+                Interface.TYPE: 'ethernet',
+                Interface.STATE: 'down',
```

Comment:
`ethernet` and `down` are constants as well. But fix it here only if you'll need to rebase. (you can fix it in a following patch)